Civi/Api4/Generic/BasicGetFieldsAction.php

```diff
@@ -254,10 +254,30 @@ public function fields() {
       [
         'name' => 'data_type',
         'data_type' => 'String',
+        'options' => [
```

Comment:
So we also have this function for custom fields....

https://github.com/civicrm/civicrm-core/blob/fe41c91d5b639cb1f2946b05420d6e2d86a79a91/CRM/Core/BAO/CustomField.php#L36

And a sort of pattern of adding lists like this to CRM_Core_SelectValues

so I'm wondering if this could fit with either of those

Reply:
Well, the APIv4 is a little different from that. It works to normalize the differences between custom field types and core field types (which are inexplicably inconsistent with each other) so it has its own list of field types. This PR is just aimed to make that more clear at the API level, so IMO placing the code in the `BasicGetFieldsAction` class is fine.

Reply:
OK